CONTRIBUTING.md

```diff
@@ -21,6 +21,25 @@ Please submit change requests and / or features via [Issues](https://github.com/
 
 All resource and data source changes must be reflected in the documentation, including new attributes and changes to existing attributes. The documentation is generated via `make docs`.  For more detailed instructions for updating docs, see ["Documenting the provider" in the development guide](./DEVELOPMENT.md#documenting-the-provider).
 
+## Pull Requests
+All command changes must be reflected in the documentation, this includes new commands and changes to options. To generate the updated documentation, first run `make docs` and confirm that your documentation changes render correctly. Then if it looks good, proceed to run `git` for the Pull Request.
```

Comment:
This text already exists on line 22 above

Reply:
I'd recommend keeping the existing text, but adding the `## Pull Requests` header above that so it more clearly applies to PRs

Reply:
I applied these changes through the PR review suggestions.